app/services/license_checkout_service.rb

```diff
@@ -11,6 +11,7 @@ class InvalidLicenseError < StandardError; end
     policy
     group
     owner
+    users
```

Comment:
It would be nice to add a limit to how many users (and entitlements) can be included.